### ISSUE TITLE
should have a config for receive.replication-factor params

### ISSUE THREAD

**Issue by songleo** [observatorium/operator#26]
I check observatorium CRD, but I can not found there has a config to modify **receive.replication-factor** , I can modify this setting in the statefulsets post install. but I expected the CR setting to already do that. this option does not exist in the operator today. thanks.

![image](https://user-images.githubusercontent.com/6408381/99142137-3e283a00-268d-11eb-8606-218494eb7a8a.png)

@squat FYI

**Comment by squat** [comment]
Yes this is the state of the world today. We are super happy to accept a little request adding this feature 🎉
